config/database.yml.example

```diff
@@ -1,22 +0,0 @@
-default: &default
```

Comment:
Why are we deleting this?

Reply:
I changed the filename from `database.yml.example` to `database.yml` cos it was giving me troubles when I pulled and merge from master back then.

Reply:
You should see the `README` haha, when you install need to copy. This needs to be here so that people can copy basically

Reply:
Oppps. Okie. It just got added back in.